src/main/java/com/smiley/smileybackend/dto/user/MedicalInfo.java

```diff
@@ -0,0 +1,47 @@
+package com.smiley.smileybackend.dto.user;
+
+import com.smiley.smileybackend.domain.Hospital;
+import com.smiley.smileybackend.domain.User;
+import com.smiley.smileybackend.domain.UserMedicalInfo;
+import io.swagger.annotations.ApiModelProperty;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.ToString;
+import javax.validation.constraints.NotNull;
+import java.time.LocalDate;
+import java.util.List;
+
+@Getter
+@ToString
+@AllArgsConstructor
+@NoArgsConstructor
+public class MedicalInfo {
+    @NotNull
+    @ApiModelProperty(value = "사용자 ID(user의 index번호). 공백 X")
+    private Integer user_id;
```

Comment:
변수 네이밍 규칙 안맞는 부분 수정 부탁드립니다 !
```suggestion
    private Integer userId;
```

Reply:
위에서 말씀하신 부분만 수정하면 될 것 같습니다.